README.md

```diff
@@ -146,4 +135,5 @@ License
  See the License for the specific language governing permissions and
  limitations under the License.
  ```
-Copyright (c) 2014, 2016 by Delphix. All rights reserved.
+
+Copyright (c) 2014, 2022 by Delphix. All rights reserved.
```

Comment:
I think this could go on top same as in other files

Reply:
It comes as a very large heading in README. Do we follow the same for all other README's too ?

Reply:
DB2 adds it but as a comment like below

`<!--
    Copyright (c) 2021, 2022 by Delphix. All rights reserved.
-->`

Reply:
Followed terraform and mySQL and removed the copyright.